src/data/AboutItems.json

```
@@ -0,0 +1,48 @@
+{
+	"menu": [
+		{
+			"title": "GitHub",
+			"icon": "logo-github",
+			"content": "Check out our source code",
+			"url": "https://github.com/SHPEUCF/shpeucfapp"
+		},
+		{
+			"title": "Bugs",
+			"icon": "ios-bug",
+			"content": "Find any bugs? Submit them here",
+			"url": "https://github.com/SHPEUCF/shpeucfapp/issues"
+		},
+		{
+			"title": "Privacy Policy",
+			"icon": "ios-document",
+			"url": "https://www.shpeucf.com/privacy-policy/"
+		},
+		{
+			"title": "Change Log",
+			"icon": "md-checkmark-circle-outline",
+			"url": "https://github.com/SHPEUCF/shpeucfapp/releases"
+		},
+		{
+			"title": "MIT License",
+			"icon": "ios-paper",
+			"url": "https://github.com/SHPEUCF/shpeucfapp/blob/master/LICENSE"
+		}
+	],
+	"developers": [
+		{
+			"name": "Luis Benavides",
+			"pic": "https://avatars3.githubusercontent.com/u/3769480?s=400&u=b4264005e8109ad5ceac231a07bac6501ea7e6e8&v=4",
+			"github": "https://github.com/luisbenan"
+		},
+		{
+			"name": "Haniel Diaz",
+			"pic": "https://avatars1.githubusercontent.com/u/22435531?s=460&u=bca76a4a97203410f7fb33a07a8343051d2aa586&v=4",
+			"github": "https://github.com/HanielDiaz"
+		},
+		{
+			"name": "Steven Perdomo",
+			"pic": "https://avatars3.githubusercontent.com/u/44217113?s=400&u=1a8087bd125d9c123b53248e33f56230e6ea70bb&v=4",
+			"github": "https://github.com/esteban737"
+		}
+	]
```

You should think about making separate variables. You don't need to make this a .json file. You can just export objects from here in a .js file. This will allow you to make better names for the lists when you import them.
```suggestion
	export developers = [data]
	export menuItems = [data]
	
	import { developers, menuItems } from 'whatfile.js'
```